### PR TITLE
Upload file support for OpenApiProvider

### PR DESCRIPTION
**Pull request by sergey-tihon**
Issue 

>  Operation 'uploadFile' does not contain supported media types ["application/octet-stream"]

### CODE REVIEW

**Comment by sergey-tihon** [comment]
Releases in [v2.0.0-beta5](https://www.nuget.org/packages/SwaggerProvider/2.0.0-beta5)

File: tests/SwaggerProvider.ProviderTests/Schemas/v3/petstore.yaml [hunk 1]
```diff
@@ -1,109 +1,962 @@
...
-        - code
-        - message
-      properties:
-        code:
-          type: integer
-          format: int32
-        message:
-          type: string
\ No newline at end of file
+{
```
Comment (xperiandri) [inline tests/SwaggerProvider.ProviderTests/Schemas/v3/petstore.yaml:1]: Interesting, why it is now JSON instead of YAML?

Comment (sergey-tihon) [reply tests/SwaggerProvider.ProviderTests/Schemas/v3/petstore.yaml:1]: no, both works

Comment (xperiandri) [reply tests/SwaggerProvider.ProviderTests/Schemas/v3/petstore.yaml:1]: Yes, but then file must be named `.json` 🙂
And YAML is more readable...

**Pull request merged**